fuzzers/run_fuzzer.py

```diff
@@ -287,6 +287,9 @@ def mem_convert(s):
     62.0
     """
     units = {
+        'Gi': pow(2, 30),
+        'Mi': pow(2, 20),
+        'Ki': pow(2, 10),
```

Comment:
Minor nit - feel free not to change.

I find the `pow(2, 10)` form a bit weird. I would have written `1024`, `1024*1024` and `1024*1024*1024` personally.